doc/faq.md

```diff
@@ -64,6 +64,39 @@ Haskell's `aeson` library encodes long integers as JSON numbers, which leads to
 
 ### Q: I use wayland, the E2E browser fails on startup
 
+We are aware of two error messages that can be show to you if you are using wayland.
+
+<details>
+  <summary>You can get something like this if you try to open the e2e browser</summary>
+
+  10 silly lifecycle cardano-transaction-lib@3.0.0~e2e-browser: Args: [
```

Comment:
This does not render properly, use code blocks inside.

https://github.com/Plutonomicon/cardano-transaction-lib/blob/4c82c3870a10fffa4e9f27e69fc3bd7017fb43aa/doc/faq.md

Reply:
it's already fixed
https://github.com/Plutonomicon/cardano-transaction-lib/blob/cf0ade4f9d8daf3ebb2c7bcd08ea4760076cd6a5/doc/faq.md

Reply:
Thanks, could you please remove the log levels too (`silly lifecycle` and friends)